### PR TITLE
(rich-text-editor) - addition of some new tokens

### PR DESCRIPTION
**Pull request by gavin-everett-genesys**

### CODE REVIEW

**Comment by gavin-everett-genesys** [comment]
Tokens branch is now ready for review 👍

**Comment by 321gillian** [comment]
Can you update with main again?

File: packages/genesys-spark-components/src/components/beta/gux-rich-text-editor/gux-rich-text-editor.light.scss [hunk 1]
```diff
@@ -32,9 +33,8 @@
     color: var(--gse-ui-rte-codeBlock-foregroundColor);
     white-space: pre-wrap;
     background-color: var(--gse-ui-rte-codeBlock-backgroundColor);
-    border: var(--gse-ui-rte-codeBlock-border-width)
-      var(--gse-ui-rte-codeBlock-border-style)
-      var(--gse-ui-rte-codeBlock-border-color);
-    border-radius: 4px; // Token Required : COMUI-3190
+    border: var(--gse-ui-rte-codeBlock-border-width) var(--gse-ui-rte-codeBlock-border-style)
+      (--gse-ui-rte-codeblock-border-color);
```
Comment (daragh-king-genesys) [inline packages/genesys-spark-components/src/components/beta/gux-rich-text-editor/gux-rich-text-editor.light.scss:37]: Typo `--gse-ui-rte-codeblock-border-color` -> `--gse-ui-rte-codeBlock-border-color`

File: packages/genesys-spark-tokens/reports/used-undefined-tokens.json [hunk 2]
```diff
@@ -1 +1 @@
-[]
+["--gse-ui-rte-codeblock-border-color"]
```
Comment (daragh-king-genesys) [inline packages/genesys-spark-tokens/reports/used-undefined-tokens.json:1]: This is how I found the typo :)

Comment (gavin-everett-genesys) [reply packages/genesys-spark-tokens/reports/used-undefined-tokens.json]: Resolved now 👍

**Review by conor-darcy — APPROVED** [review]

**Pull request merged**